### PR TITLE
parity_check, quasi-twisted, pless_symmetry

### PR DESCRIPTION
**Pull request by cbclayton**

### CODE REVIEW

File: codes/classical/q-ary_digits/pless_symmetry.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  Pless symmetry codes have the interesting property that they are \emph{self-dual}.
+  This is due to the fact that \(S_qS_q^T=qI\), and therefore \(S_qS_q^T=-I\) (mod 3).
+
+
+notes:
+  - 'Pless symmetry codes are ternary codes.'
+
+relations:
+  parents:
+    - code_id: q-ary_linear
```
Comment (AlexanderBarg) [inline codes/classical/q-ary_digits/pless_symmetry.yml:37]: q-ary Self-dual codes is also a parent

Comment (AlexanderBarg) [reply codes/classical/q-ary_digits/pless_symmetry.yml:37]: Cousins: Quadratic-residue codes, duadic codes

File: codes/classical/properties/q-ary_self_dual.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#######################################################
```
Comment (valbert4) [inline codes/classical/properties/q-ary_self_dual.yml:1]: No need for this; just point to dual when referenced.

File: codes/classical/q-ary_digits/pless_symmetry.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    \includegraphics{pless_symmetry_table.svg}
+  \end{table}
+
+  where \(\chi(0)=0\), \(\chi(x)=1\) if \(x\) is a square in \(GF(q)\) and, \(\chi(x)=-1\) if \(x\) is a not square in \(GF(q)\).
+
+  Pless symmetry codes have the interesting property that they are \emph{self-dual}.
+  This is due to the fact that \(S_qS_q^T=qI\), and therefore \(S_qS_q^T=-I\) (mod 3).
+
+
+notes:
```
Comment (valbert4) [inline codes/classical/q-ary_digits/pless_symmetry.yml:32]: Can remove notes.

File: codes/classical/q-ary_digits/pless_symmetry.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  - 'Pless symmetry codes are ternary codes.'
+
+relations:
+  parents:
+    - code_id: q-ary_linear
+      detail: ''
+    - code_id: q-ary_self_dual
```
Comment (valbert4) [inline codes/classical/q-ary_digits/pless_symmetry.yml:39]: Point to code_id:dual and mention in comment the relevance.

File: codes/classical/q-ary_digits/quasi_twisted.yml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  Then the code \(C\) is \((\lambda,l)\)-quasi-twisted\cite{doi:10.1016/j.ffa.2011.08.001} if for any codeword \((c_1, \dots, c_n)\in C\),
+  \begin{align*}
+    (\lambda c_{n-l+1}, \lambda c_{n-l+2}, \dots, \lambda c_{n}, c_1, \dots, c_{n-l})\in C.
+  \end{align*}
+  Quasi-cyclic codes are a special case of quasi-twisted codes where \(\lambda=1\),
+  and constacyclic codes are a special case where \(l=1\).
+
+relations:
+  parents:
+    - code_id: q-ary_linear
```
Comment (valbert4) [inline codes/classical/q-ary_digits/quasi_twisted.yml:25]: ecc

**Pull request closed without merging**